bloges56/DateSim
# Rhytm level1 gray box

## Description
- Create the graybox/initial design for the rhythm game.

## How to Test
1. Make sure that the RhythmLevelOne scene is added in the build settings. 
2. Go to SceneManager Scene and hit play
3. Hit start game
4. Click W,A,S or D depending on what is prompted by the box
5. If the game doesn't start after clicking the Start Game button , make sure the following settings are correct in the HomeScreenManager object's scripts settings in the HomeScreen scene: 
![Setting](https://user-images.githubusercontent.com/60618398/162842031-9d7e98d8-750d-4574-a197-db998b91689e.png)



## Checklist
- [x] Does the game start
- [x] Does clicking the corresponding key make the key green? 
- [x] Does clicking the incorrect key(that is still W,A,S,D) make the key red? 
- [x] Does the timer count down when the level starts?
- [x] Does the amount of lives go down when you click an incorrect key?
- [x] Does the current score update when you click the correct key?
- [x] Does the high score update when you beat it?
- [x] Does the game end when the time runs out?
- [x] Does the game end when the lives run out?
- [x] Does the game show the winning screen when you beat the high score?
- [x] Does the game show the loosing screen when the game finishes without beating the high score?
- [x] Does the HomeScreen pop up after the winning/loosing screen shows up? 

## Usage
1. To start the game on your desired scene, do the following:
2. In the HomeScreen scene, click the HomeScreenManager object. 
3. In the script section, select the Override checkbox and input a scene name to load. Make sure the scene is in the build settings.
![Setting](https://user-images.githubusercontent.com/60618398/162842031-9d7e98d8-750d-4574-a197-db998b91689e.png)
